packages/perseus-core/src/utils/util.ts

```diff
@@ -21,6 +21,54 @@ const seededRNG: (seed: number) => RNG = function (seed: number): RNG {
     };
 };
 
+// Note: Previously called cyrb128
+function getHashes(str: string): [number, number, number, number] {
+    let h1 = 1779033703;
+    let h2 = 3144134277;
+    let h3 = 1013904242;
+    let h4 = 2773480762;
+    let k = 0;
+    for (let i = 0; i < str.length; i++) {
+        k = str.charCodeAt(i);
+        h1 = h2 ^ Math.imul(h1 ^ k, 597399067);
+        h2 = h3 ^ Math.imul(h2 ^ k, 2869860233);
+        h3 = h4 ^ Math.imul(h3 ^ k, 951274213);
+        h4 = h1 ^ Math.imul(h4 ^ k, 2716044179);
+    }
+    h1 = Math.imul(h3 ^ (h1 >>> 18), 597399067);
+    h2 = Math.imul(h4 ^ (h2 >>> 22), 2869860233);
+    h3 = Math.imul(h1 ^ (h3 >>> 17), 951274213);
+    h4 = Math.imul(h2 ^ (h4 >>> 19), 2716044179);
+    return [
+        (h1 ^ h2 ^ h3 ^ h4) >>> 0,
+        (h2 ^ h1) >>> 0,
+        (h3 ^ h1) >>> 0,
+        (h4 ^ h1) >>> 0,
+    ];
+}
+
+// Note: Previously called sfc32
+function getSeedFunctionFromHashes(
```

Comment:
Since this function returns a random number generator, could it be called something like `createRandomNumberGenerator` or `seedRandomNumberGenerator`?

I don't think I'd draw the correct conclusions from the name `getSeedFunction` if I saw this being used.

Reply:
Updated to `createRandomNumberGenerator`